Psychtoolbox/PsychHardware/NetStation.m

```diff
@@ -152,6 +178,10 @@
                 rep=receive(NSIDENTIFIER,1);
                 WaitSecs(.5);
                 pnet( NSIDENTIFIER, 'close' );
+                if GETNTPSYNCED
```

Comment:
Probably makes more sense to test for ~isempty(NTPSOCK), as that is what you want to release if it is there, regardless what might happen to GETNTPSYNCED, e.g., if a user calls the old syncrhonize method after callint the new one, however pointless that would be.